source/ref_trainings.rst

```diff
@@ -0,0 +1,269 @@
+######################
+Model Trainings
```

Comment:
Please rename it as just `Trainings` and `Model Deployments` as  just `Deployments`
OR
Rename `Packagers` to `Model Packagers` 

for better consistency.